### DISCUSSION TITLE
Can stenographer pickup pcaps from windows 10/11?

### DISCUSSION THREAD

**Discussion by iqworks** [Security-Onion-Solutions/securityonion#10484] [Unsupported Versions]
Hi, from what i am reading it looks like steno only captures data from within the SOC?

thx

**Comment by robbiemarshall** [comment]
Stenographer only ingests pcaps from the network monitoring interface.

**Reply by iqworks** [reply]
Thanks for replying. Does that mean the VM or SOC?

**Reply by robbiemarshall** [reply]
It's from whichever interface you configured during the setup process to be your monitor interface, as opposed to the management interface.
